app/views/home/subscriptions.html.erb

```diff
@@ -71,7 +71,26 @@
           <a rel="tooltip" title="Once per day" class="btn btn-default btn-xs" href="javascript:void(0)"><i class="fa fa-list"></i> Digest</a>
           -->
           <script> input_<%= subscription.tid %> = '#share-<%= subscription.tid %>'</script>
-          <a class="btn btn-outline-secondary btn-sm text-body" rel="popover"  data-toggle="popover" data-placement="left" title="<%= translation('home.subscriptions.share_subscription_link',{},false) %>" data-html="true" data-content="<div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/intent/tweet?text=🎈 Public Lab: Subscriptions&url=https://publiclab.org/subscriptions/tag/<%= subscription.tagname %>' class='twitter-share-button'>Tweet</a></div> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='//platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');</script>
+          <a class="btn btn-outline-secondary btn-sm text-body" rel="popover"  data-toggle="popover" data-placement="left" title="<%= translation('home.subscriptions.share_subscription_link',{},false) %>" data-html="true" data-content="<div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/intent/tweet?text=🎈 Public Lab: Subscriptions&url=https://publiclab.org/subscriptions/tag/<%= subscription.tagname %>' class='twitter-share-button'>Tweet</a></div>
+          <script>
+          window.twttr = (function(d, s, id) 
+          { var js, fjs = d.getElementsByTagName(s)[0],
+            t = window.twttr || {};
+            console.log(t);
```

Comment:
Also, I think this line is supposed to be `if (d.getElementById(id)) return t;`

Reply:
Oops! I forgot to remove the console line. I'll remove it!
Also `if (d.getElementById(id)) return t;` this line was the reason for the twitter-button error, it wasn't creating a new element  (twitter button) with new id for the next popovers. Only the first clicked popover showed the button.